### PR TITLE
Home task HTML_02 

### PR DESCRIPTION
**Pull request by IrynaSafonava**
Good day Siarhei,

I didn't cope with SVG task as it requires additional time to look into it. If I do the task some time later, can I send it with another home tasks ?

Thank you !

### CODE REVIEW

**Comment by pluhin** [comment]
> Good day Siarhei,
> 
> I didn't cope with SVG task as it requires additional time to look into it. If I do the task some time later, can I send it with another home tasks ?
> 
> Thank you !

Sure, why not :)

File: isafonova/HTML_01/extra_task_HTML.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset="UTF-8">
+		<title>Домашнее задание - дополнительное</title>
+	</head>
+	<body>
+		<h1 align = "center">Заголовок первого уровня</1h>
```
Comment (pluhin) [inline isafonova/HTML_01/extra_task_HTML.html]: 1h ?

File: isafonova/HTML_01/task_Python_2.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+		<code>print len(months)</code>
+		<p>которая вернёт 12.</p>
+		<p>Ещё один пример списков — категории в поваренной книге. Например...</p>
+		<code>categories = ['Main dish','Meat','Fish','Soup','Cookies']</code>
+		<p>Таким образом, в <strong>categories[0]</strong> хранится '<i>Main dish</i>', а в <strong>categories[4]</strong> — '<i>Cookies</i>'. Всё очень просто. Я уверен, вы сможете придумать много других примеров использования списков.</p>
+		<hr>
+		<h2>Комментарии</h2>
+		<p>Далее, мы должны обсудить комментарии. Комментарии важны по нескольким причинам. Они не только говорят вам или другим людям, что вы пытаетесь делать, но и напомнят вам через полгода, что же вы хотели сделать. Когда вы будете писать много программ, комментарии станут важными. Также при их помощи вы заставите Python игнорировать какие-либо части кода. Чтобы закомментировать строку, поставьте перед ней знак '#'. Например:</p>
+		#
```
Comment (pluhin) [inline isafonova/HTML_01/task_Python_2.html]: need tag pre for this char

File: isafonova/HTML_02/site.by/about.html [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+		<p><a href="index.html">Главная</a> | <a href="catalog.html">Каталог</a> | <a href="promo.html">Акции</a> | <a href="about.html">О нас</a> |</p>
+		<h2>Меню список</h2>
+		<ul>
+			<li><a href="index.html">Главная</a></li>
+			<li><a href="catalog.html">Каталог</a></li>
+			<li><a href="promo.html">Акции</a></li>
+			<li><a href="about.html">О нас</a></li>
+		</ul>
+		<h2>Меню с вложенным списком</h2>
+		<ul>
```
Comment (pluhin) [inline isafonova/HTML_02/site.by/about.html:19]: Wrong structure for the nested lists,
Could you please check all files

**Review by pluhin — CHANGES_REQUESTED** [review]

**Comment by IrynaSafonava** [comment]
Siarhei,
I had a conflict when I made changes for my 1st home task. I tried to Resolve conflict via web-interface deleting old rows and left the new onces and then clicked [Marcked as resolved] and then [Merge commit]... Is this correct way to solve conflicts or  should I do it usig some commands in Git Bash ?
Thank you !
P.S. Moreover, should I go further with it and do more actions ?

**Review by pluhin — APPROVED** [review]
Good, accepted

**Pull request merged**